components/MatesList/MatesList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Mate } from './Mate';
+import useSWR from 'swr';
+import { fetcher } from '@/lib/fetcher';
+
+export const MatesList = () => {
+  const url = `/api/user/all`;
+  const { data, error } = useSWR(url, fetcher);
```

Comment:
here you can get loading state... just display plain text stating 'loading'. later we can improve it